website/docs/tutorials/deploying-java-app-with-database.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Deploying a Java OpenLiberty application with PostgreSQL
+sidebar_position: 1
+---
+
+This scenario illustrates deploying a Java application with odo and linking it to an in-cluster PostgreSQL service in the minikube environment.
+
+In this example there are two roles:
+
+1. Cluster Admin - Prepare the cluster by installing the required operators into the cluster
+2. Application Developer - Imports a Java application, creates a Database instance, and connect the application to the Database instance.
+
+## Cluster admin
+---
+
+This section assumes that you have installed minikube and configured it. See [Getting Started > Cluster Setup > Kubernetes](../getting-started/cluster-setup/kubernetes.md).
+
+The cluster admin must install two Operators into the cluster:
+
+1. Operator Backed Service
+2. Service Binding Operator
+
+An _Operator Backed Service_ is an operator that helps in deploying instances of a given service, for example PostgreSQL, MySQL, Redis.
+
+Furthermore, these operators are "bind-able" as they expose information necessary for an application to connect to their instances.
+
+We will use Dev4Devs PostgreSQL Operator found in the [OperatorHub](https://operatorhub.io) to demonstrate a sample use case.
+
+Service Binding Operator is an operator that helps in easily binding an application to other Operator Backed Services. It accomplishes this through automatically collecting binding information and sharing with an application to bind it with operator managed backing services
+
+### Installing the Operator Backed Service
+
+* Run the following `kubectl` command to make the PostgreSQL Operator available in `my-postgresql-operator-dev4devs-com` namespace of your minikube cluster:
+```shell
+  $ kubectl create -f https://operatorhub.io/install/postgresql-operator-dev4devs-com.yaml
+  ```
+
+**Note**: The `my-postgresql-operator-dev4devs-com` Operator will be installed in the `my-postgresql-operator-dev4devs-com` namespace and will be usable from this namespace only.
+
+### Installing the Service Binding Operator
```

Comment:
I wonder if we will need a separate section on installing SBO and PostgreSQLO if we already point to the installation guides in the section above.

Reply:
No we don't. We should add a link to SBO installation that we plan to cover in kubernetes and openshift installation guides in "Getting Started" section.